.github/workflows/spack-test.yaml

```
@@ -37,8 +42,8 @@ jobs:
         run: |
           docker run -v ${{ github.workspace }}:/work ghcr.io/kokkos/kokkos-fft/base_${{ matrix.backend.image }}:latest \
           bash -c "source spack/share/spack/setup-env.sh && \
+            export SPACK_USER_CONFIG_PATH=${{ matrix.backend.spack_path }} && \
             spack install kokkos-fft +tests ^kokkos ${{ matrix.backend.kokkos }} && \
```

Can you try this ?
```suggestion
          bash -c "source spack/share/spack/setup-env.sh && \
            export SPACK_USER_CONFIG_PATH=${{ matrix.backend.spack_path }} && \
            spack compiler find && \
            spack install kokkos-fft +tests ^kokkos ${{ matrix.backend.kokkos }} && \
```

I suspect Spack to be confused with the existing packages.yaml

---

Thanks. I will try that